### DISCUSSION TITLE
robots.txt is killed by google ;)

### DISCUSSION THREAD

**Discussion by Juuhz** [frontendbr/forum#1479] [Dicas]
Fala Devs,

Estou abrindo esse tópico apenas para compartilhar com vocês a nova feature do Google, que impacta indiretamente, nós, front-ends. 

Google irá acabar com o arquivo robots, então se seu site utiliza ele, você terá que passar as regras descritas nele para o `<head>` das suas páginas utilizando `<metatag>`.

Segue o artigo:
https://webmasters.googleblog.com/2019/07/a-note-on-unsupported-rules-in-robotstxt.html

Abraços.

**Comment by felipefialho** [comment]
Valeu a dica @Juuhz, não tava sabendo dessa.

**Comment by mariorodeghiero** [comment]
valeu @Juuhz

**Comment by kvnol** [comment]
boa!
